assets/scripts/config.js

```diff
@@ -15,6 +15,27 @@ function loadFormByData(){
   }
 }
 
+let hoursInputs = document.querySelectorAll("#work-time, #break-time")
```

Comment:
To follow the design pattern, it would be interesting for us to encapsulate these new instructions in a function called in the controller event of this file (on line 1) which would be the DOMContentLoaded event.

something like:

```js
function handleTimeTnputs(){
  // your new code here..
}
```
That way, we would run this new function right after the `handleSubmit()` statement on line 3.

```js
window.addEventListener('DOMContentLoaded', () => {
  loadFormByData();
  handleSubmit();
  handleTimeTnputs(); // new instruction
});
```